webform_strawberryfield.routing.yml

```diff
@@ -20,6 +20,14 @@ webform_strawberryfield.auth_autocomplete:
     _access: 'TRUE'
 webform_strawberryfield.nominatim:
   path: '/webform_strawberry/nominatim/{api_type}/{count}/{lang}'
+  defaults:
+    _controller: '\Drupal\webform_strawberryfield\Controller\NominatimController::handleRequest'
+    _format: json
+  requirements:
+    _permission: 'access content'
+    _csrf_token: 'TRUE'
+webform_strawberryfield.nominatim_reverse:
+  path: '/webform_strawberry/nominatim/{api_type}/{lang}'
```

Comment:
We could do this simpler. Both paths are quite similar, too similar. And both use` {api_type}` and call the same method in the controller. So i wonder if it would be clearer if, we just keep a single one, document in a comment what {api_type} can be (two options), give it a default (see https://symfony.com/doc/current/routing.html#optional-parameters, basically what i just added as a suggestion there, and then simply deal with, ignoring count for the case of reverse and inline documenting the fact that reverse always returns a single one.
To be honest, i don't see anything bad there in your routes, but thinking about how APIs work, having two different routes for the mostly the same functionality is kinda weird, and we could, by mistake allowing people (yeah, people like to do stuff) to call one (reverse) with the search value in {api_type} which will lead to a search on a reverse path without a count... not sure if i explain myself =)